trl/trainer/ppo_trainer.py

```diff
@@ -41,6 +42,42 @@
 from trl.trainer.utils import AdaptiveKLController, FixedKLController
 
 
+MODEL_CARD_TEMPLATE = """---
+license: apache-2.0
+tags:
+- trl
+- transformers
+- reinforcement-learning
+- human-feedback
```

Comment:
maybe `human-feedback` should be added by the user manually, since you don't necessarily need to train from human feedback (e.g. all the sentiment examples).

Reply:
Fixed in https://github.com/lvwerra/trl/pull/68/commits/47289f5bf3b20df8a5870478dc68fd6032af20ad